conf/script/ha_setup

```diff
@@ -39,8 +39,11 @@ post_install() {
     mkdir -p /var/log/seagate/cortx/ha/
     chown -R csm /var/log/seagate/cortx/ha/
```

Comment:
https://github.com/Seagate/cortx-ha/tree/main/conf/etc/cron.hourly

Do we need this dir if not can we remove it

Reply:
done